src/bp-moderation/classes/class-bp-moderation-activity-comment.php

```diff
@@ -72,13 +72,39 @@ public static function get_content_owner_id( $activity_id ) {
 	 * @since BuddyBoss 2.0.0
 	 *
 	 * @param int $activity_comment_id activity id.
+	 * @param bool $view_link   add view link
 	 *
 	 * @return string
 	 */
-	public static function get_content_excerpt( $activity_comment_id ) {
+	public static function get_content_excerpt( $activity_comment_id, $view_link ) {
```

Comment:
@janakkaneriya007 `view_link ` default value should be `false` in all content type class